src/core/env.js

```diff
@@ -113,15 +113,15 @@ const defaults = new Map(
       type: "string",
       default: "https://cfstore.rethinkdns.com/blocklists/",
     },
-    // primary doh upstream
+    // primary doh or doh3 upstream
     CF_DNS_RESOLVER_URL: {
       type: "string",
-      default: "https://cloudflare-dns.com/dns-query",
+      default: "h3://cloudflare-dns.com/dns-query",
```

Comment:
`h3://` isn't a valid protocol scheme. Even if it works, it isn't doing what you think it might be. AdGuard has some funky representation for DNS URLs, but those aren't universally adopted or standardized.

For HTTP3, the scheme remains `https://`. The upgrade to HTTP3 happens with `alt-svc` hints (via headers), or https/svcb hints (via dns), afaik.

Reply:
Ok, I switched back that change and now only requesting updating the readme for the cloudflare location amount change